xray/test/test_dataarray.py

```diff
@@ -1276,15 +1276,15 @@ def test_broadcast_arrays(self):
 
         x = DataArray(np.random.randn(2, 3), dims=['a', 'b'])
         y = DataArray(np.random.randn(3, 2), dims=['b', 'a'])
-        x2, y2 = broadcast_arrays(x, y)
+        x2, y2 = broadcast(x, y)
```

Comment:
Can there be a test for broadcasting coordinates, not just data variables? That is something I use frequently, and I currently have to hack it (as in #649).